core/imageroot/usr/local/agent/bin/module-backup

```diff
@@ -114,20 +137,18 @@ try:
     agent.run_restic(rdb, repository, repopath, [], ["forget", "--prune", "--keep-last=" + obackup['retention']]).check_returncode()
 
 except subprocess.CalledProcessError as ex:
-    errors = 1
+    print("[ERROR] restic backup failed.", ex, file=sys.stderr)
+    errors += 1
```

Comment:
Replace with:
    backup['errors'] += 1

Reply:
I would prefer to stick to errors if you could accept